scene/gui/button.h

```diff
@@ -75,6 +75,8 @@ class Button : public BaseButton {
 		float style_margin_top = 0;
 		float style_margin_bottom = 0;
 
+		int align_to_largest_stylebox = 0;
```

Comment:
```suggestion
		bool align_to_largest_stylebox = false;
```
The cache value can be a bool.

Reply:
Yes, this seems to work. Updated PR.

I guess other "boolean" theme constants, like `Tree::draw_relationship_lines` and `Tree::draw_guides`, should be changed as well. For 4.4 we should probably allow different theme constant types (I have support drafted in https://github.com/godotengine/godot/pull/87243#issuecomment-1894266187, but it's a breaking change).

Reply:
> I guess other "boolean" theme constants, like Tree::draw_relationship_lines and Tree::draw_guides, should be changed as well.

Yeah, we already use bool constants in some classes, but not everywhere.